build.gradle

```diff
@@ -0,0 +1,354 @@
+import javax.tools.ToolProvider
+import org.ajoberstar.grgit.*
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id "java"
+    id 'maven'
+    id 'signing'
+    id 'jacoco'
+    id 'com.palantir.git-version' version '0.5.1'
+    id 'com.github.johnrengelman.shadow' version '1.2.3'
+    id "com.github.kt3k.coveralls" version "2.6.3"
+    id 'org.ajoberstar.grgit' version "1.4.2"
+}
+
+repositories {
+    mavenCentral()
+
+    // Include the Broad artifactory for intel-gkl snapshots. This is temporary
+    // until intel-gkl is available on maven central.
+    //maven {
+    //    url "https://artifactory.broadinstitute.org/artifactory/libs-snapshot/"
+    //}
```

Comment:
Should probably remove this commented-out reference to the artifactory, since `intel-gkl` is not currently a Picard dependency.